INPTPZ1/ComplexNumber.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+namespace INPTPZ1
+{
+
+    namespace Mathematics
+    {
+        public class ComplexNumber
+        {
+            public double RealPart { get; set; }
+            public double ImaginaryPart { get; set; }
+
+            public override bool Equals(object newItem)
+            {
+                if (newItem is ComplexNumber)
+                {
+                    ComplexNumber complexNumber = newItem as ComplexNumber;
+                    return complexNumber.RealPart == RealPart && complexNumber.ImaginaryPart == ImaginaryPart;
+                }
+                return base.Equals(newItem);
+            }
+
+            public readonly static ComplexNumber zeroComplexNumber = new ComplexNumber()
```

Comment:
please reorder your class properly - constants, attributes, ..., methods; and do not mix it
also you are not following official naming guide for .NET